zerver/openapi/zulip.yaml

```diff
@@ -22553,6 +22553,231 @@ paths:
       responses:
         "200":
           $ref: "#/components/responses/SimpleSuccess"
+
+  /channels/create:
+    post:
+      tags: ["channels"]
+      operationId: create-channel
+      summary: Create a channel
+      description: |
+        Create a new [channel](/help/create-channels) and optionally
+        subscribe users to the newly created channel. The initial [channel settings](/api/update-stream)
+        will be determined by the optional parameters, like `invite_only`, detailed below.
+
+        Note that the ability to subscribe oneself and/or other users to a specified
+        channel depends on the [channel's type: private, public, or web-public](/help/channel-permissions).
+        **Changes**: New in Zulip 11.0 (feature level 399).
```

Comment:
I think we should also mention here about how the channels were created previously.